web/src/portico/landing-page.ts

```diff
@@ -94,12 +137,27 @@ const apps_events = function () {
 
         $(".info .platform").text(version_info.alt);
         $(".info .description").html(version_info.description);
-        $desktop_download_link.attr("href", version_info.download_link);
-        $download_from_google_play_store.attr("href", version_info.play_store_link);
-        $download_from_apple_app_store.attr("href", version_info.app_store_link);
-        $download_android_apk.find("a").attr("href", version_info.download_link);
-        $download_mac_arm64.find("a").attr("href", version_info.mac_arm64_link);
-        $download_instructions.html(version_info.download_instructions);
+
+        if (version_info.alt === "Android") {
+            $download_from_google_play_store.attr("href", version_info.play_store_link);
+            $download_android_apk.find("a").attr("href", version_info.download_link);
+        }
+        if (version_info.alt === "iOS") {
+            $download_from_apple_app_store.attr("href", version_info.app_store_link);
+        }
+        if (version_info.alt === "macOS") {
+            $download_mac_arm64.find("a").attr("href", version_info.mac_arm64_link);
+        }
+        if (
+            version_info.alt === "Windows" ||
+            version_info.alt === "Linux" ||
+            version_info.alt === "macOS"
+        ) {
+            $desktop_download_link.attr("href", version_info.download_link);
+        }
```

Comment:
It seems like this one could check the presence of `download_link` in an `else` clause to the `version_info.play_store_link` for Android, and avoid the highly hardcoded implementation?

Reply:
You can also only define values like `$download_from_google_play_store` inside the blocks that'll use them.

Reply:
I did a bit of refactoring to write the logic in a cleaner way